terraform/groups/ecs-service/variables.tf

```diff
@@ -28,30 +28,35 @@ variable "docker_registry" {
 # Service performance and scaling configs
 # ------------------------------------------------------------------------------
 variable "desired_task_count" {
-  type = number
+  type        = number
   description = "The desired ECS task count for this service"
-  default = 1 # defaulted low for dev environments, override for production
+  default     = 1 # defaulted low for dev environments, override for production
 }
 variable "required_cpus" {
-  type = number
+  type        = number
   description = "The required cpu resource for this service. 1024 here is 1 vCPU"
-  default = 768 # defaulted low for dev environments, override for production
+  default     = 768 # defaulted low for dev environments, override for production
 }
 variable "required_memory" {
-  type = number
+  type        = number
   description = "The required memory for this service"
-  default = 1536 # defaulted low for node service in dev environments, override for production
+  default     = 1536 # defaulted low for node service in dev environments, override for production
 }
 
 variable "eric_cpus" {
-  type = number
+  type        = number
   description = "The required cpu resource for eric. 1024 here is 1 vCPU"
-  default = 256
+  default     = 256
 }
 variable "eric_memory" {
-  type = number
+  type        = number
   description = "The required memory for eric"
-  default = 512
+  default     = 512
+}
+variable "min_task_count" {
```

Comment:
Only minor, but I would probably group the desired_task_count, min_task_count and max_task_count together.